_posts/2017-05-15-rust-at-two-years.md

```diff
@@ -0,0 +1,352 @@
+
+## Rust in production
+
+In addition to the 48 new Rust friends, we now have a [Rust jobs website]! More
+and more companies are choosing Rust to solve problems involving performance,
+scaling, and safety. Let's check in on a few of them.
+
+[Rust jobs website]: http://rustjobs.rs/
+
+[Dropbox] is using Rust in multiple high-impact projects to manage exabytes of
+data where correctness and efficiency is critical. Rust is currently shipping
+in the desktop client on Windows running on hundreds of millions of machines!
+Jamie Turner recently [spoke at the SF Rust Meetup] about the details on how
+Rust helps Dropbox use less RAM and get more throughput with less CPU.
+
+[Dropbox]: https://www.dropbox.com/
+[spoke at the SF Rust Meetup]: https://air.mozilla.org/rust-meetup-may-2017/
+
+Mozilla, Rust's main sponsor, has accelerated their use of Rust in production.
+Not only did [Servo start shipping nightly builds], [Firefox 48] marked the
+first Firefox release that included Rust code as part of the [Oxidation]
+project. [Project Quantum], announced in October 2016, is an effort to
+incrementally adopt proven parts of Servo into Firefox's rendering engine,
+Gecko.
+
+[Servo start shipping nightly builds]: https://blog.servo.org/2016/06/30/servo-nightlies/
+[Firefox 48]: https://hacks.mozilla.org/2016/07/shipping-rust-in-firefox/
+[Oxidation]: https://wiki.mozilla.org/Oxidation
+[Project Quantum]: https://medium.com/mozilla-tech/a-quantum-leap-for-the-web-a3b7174b3c12
+
+[GNOME], a free and open source desktop environment for Linux, went from
+experimenting with Rust in [librsvg] in October 2016 to a [hackfest] in March
+to work on the interoperability between GNOME and Rust to enable more GNOME
+components to be written in Rust. The hackfest participants made good progress,
+be sure to check out the reports at the bottom of the hackfest page for all the
+details. We're all excited about the possibilities of Rust and GNOME working
+together!
+
+[GNOME]: https://www.gnome.org/
+[librsvg]: https://people.gnome.org/~federico/news-2016-10.html#25
+[hackfest]: https://wiki.gnome.org/Hackfests/Rust2017
+
+This year, [npm] started using Rust in production to serve JavaScript packages,
+using Rust to eliminate performance bottlenecks in their platform that serves
+around 350 million packages a day. [Ashley Williams recently gave a
+talk][ag_dubs] at RustFest in Ukraine about npm's experience with Rust in
+production; video is coming soon!
+
+[npm]: https://www.npmjs.com/
+[ag_dubs]: http://2017.rustfest.eu/talks/#how-i-convinced-the-world-s-largest-package-manager-to-use-rust-and-so-can-you
+
+## Rust in community
+
+Speaking of conferences, We've had four Rust conferences in the last year:
+
+- September 9-10, 2016: [RustConf 2016] in Portland, OR, USA;
+- September 17, 2016: [RustFest 2016] in Berlin, Germany;
+- October 27-28, 2016: [Rust Belt Rust 2016] in Pittsburgh, PA, USA;
+- April 29-30, 2017: [RustFest 2017] in Kyiv, Ukraine.
```

Comment:
I don't know why either, they work when I run jekyll locally so they should work in the published version.